### PR TITLE
chore(release): prepare for release

### PR DESCRIPTION
**Pull request by Lyokone**
## Description

*Replace this paragraph with a description of what this PR is doing. If you're modifying existing behavior, describe the existing behavior, how this PR is changing it, and what motivated the change.*

## Related Issues

*Replace this paragraph with a list of issues related to this PR from the [issue database](https://github.com/flutter/flutter/issues). Indicate, which of these issues are resolved or fixed by this PR. Note that you'll have to prefix the issue numbers with flutter/flutter#.*

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`).
This will ensure a smooth and quick review process. Updating the `pubspec.yaml` and changelogs is not required.

- [ ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [ ] All existing and new tests are passing.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] The analyzer (`melos run analyze`) does not report any problems on my PR.
- [ ] I read and followed the [Flutter Style Guide].
- [ ] I signed the [CLA].
- [ ] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change.
- [ ] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/firebase/flutterfire/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

File: packages/cloud_firestore/cloud_firestore/example/pubspec.yaml [hunk 1]
```diff
@@ -6,8 +6,8 @@ environment:
   flutter: '>=1.12.13+hotfix.4'
 
 dependencies:
-  cloud_firestore: any
-  firebase_core: any
+  cloud_firestore: ^4.3.2
```
Comment (russellwheatley) [inline packages/cloud_firestore/cloud_firestore/example/pubspec.yaml:9]: I assume this was intended?

File: packages/cloud_firestore/cloud_firestore/pubspec.yaml [hunk 2]
```diff
@@ -16,10 +16,10 @@ environment:
 
 dependencies:
   cloud_firestore_platform_interface: ^5.10.0
```
Comment (russellwheatley) [inline packages/cloud_firestore/cloud_firestore/pubspec.yaml:18]: shouldn't this bump to latest `cloud_firestore_platform_interface` version? `5.10.2`

File: packages/cloud_firestore/cloud_firestore_web/pubspec.yaml [hunk 3]
```diff
@@ -3,26 +3,26 @@ description: The web implementation of cloud_firestore
...
+version: 3.2.2
 
 environment:
   sdk: ">=2.16.0 <3.0.0"
   flutter: ">=1.20.0"
 
 dependencies:
-  _flutterfire_internals: ^1.0.12
+  _flutterfire_internals: ^1.0.13
   cloud_firestore_platform_interface: ^5.10.0
```
Comment (russellwheatley) [inline packages/cloud_firestore/cloud_firestore_web/pubspec.yaml:14]: same?

File: packages/cloud_firestore_odm/cloud_firestore_odm_generator/cloud_firestore_odm_generator_integration_test/pubspec.yaml [hunk 4]
```diff
@@ -6,8 +6,8 @@ environment:
   sdk: ">=2.16.0 <3.0.0"
 
 dependencies:
-  cloud_firestore: ^4.3.1
-  cloud_firestore_odm: any
+  cloud_firestore: ^4.3.2
```
Comment (russellwheatley) [inline packages/cloud_firestore_odm/cloud_firestore_odm_generator/cloud_firestore_odm_generator_integration_test/pubspec.yaml:9]: The `cloud_firestore` pubspec hasn't updated 🤔

**Review by russellwheatley — CHANGES_REQUESTED** [review]

File: packages/cloud_firestore/cloud_firestore/example/pubspec.yaml (hunk 1 above)
Comment (russellwheatley) [reply packages/cloud_firestore/cloud_firestore/example/pubspec.yaml:9]: referring to the switch from `any`

File: packages/cloud_firestore_odm/cloud_firestore_odm_generator/cloud_firestore_odm_generator_integration_test/pubspec.yaml (hunk 4 above)
Comment (Salakar) [reply packages/cloud_firestore_odm/cloud_firestore_odm_generator/cloud_firestore_odm_generator_integration_test/pubspec.yaml:9]: I can see it has?

Comment (russellwheatley) [reply packages/cloud_firestore_odm/cloud_firestore_odm_generator/cloud_firestore_odm_generator_integration_test/pubspec.yaml:9]: ah, my bad 😓

**Review by russellwheatley — APPROVED** [review]

**Review by lesnitsky — APPROVED** [review]

**Pull request merged**